src/convertToObject.js

```diff
@@ -13,23 +13,23 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  const stylesWithSpace = sourceString
-    .split(';')
-    .map(style => style.trim())
-    .filter(style => style.length > 0)
-    .map(style => style.split(':'));
 
-  const stylesNoSpace = stylesWithSpace.map(style =>
-    style.map(key => key.trim()));
+const callbackForReduce = (prev, element) => {
+  return {
+    ...prev,
+    [element[0]]: element[1],
+  };
+};
 
-  const styles = stylesNoSpace.reduce((prev, style) => {
-    return {
-      ...prev,
-      [style[0]]: style[1],
-    };
-  },
-  {});
+function convertToObject(sourceString) {
+  const styles = sourceString
+    .split(';')
+    .map(row => row.trim())
+    .filter(row => row.length > 0)
+    .map(item => item.split(':'))
+    .map(item =>
+      item.map(key => key.trim()))
+    .reduce((prev, element) => callbackForReduce(prev, element), {});
```

Comment:
reduce takes a callback as a parameter and start value. but you wrote that reduce takes (prev, elem)
`.reduce(callbackForReduce(prev, element), {});`